Crew Manifest Department Colors

Hubblenaut opened the pull request:
🆑 Hubblenaut
rscadd: Introduces department colors for the manifest.
tweak: The crew manifest for ghosts is now black.
/:cl:

- Departments have their respective color in the manifest.
- The manifest for ghosts is now the same as the manifest from the lobby.

Note: With the exception of Command Support, all departments use the same color code as their respective radio frequency.

<details> 
  <summary>Image</summary>

   ![grafik](https://user-images.githubusercontent.com/6383576/221308377-c210f378-f429-479e-9255-a8982def7342.png)
</details>

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

SierraKomodo commented on code/modules/mob/observer/ghost/ghost.dm:
```diff
@@ -392,15 +392,18 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	var/dat
 	dat += "<h4>Crew Manifest</h4>"
-	dat += html_crew_manifest()
+	dat += html_crew_manifest(OOC = 1)
```
```suggestion
	dat += html_crew_manifest(OOC = TRUE)
```

Hubblenaut committed:
Crew Manifest Department Colors

- Departments have their respective color in the manifest.
- The manifest for ghosts is now the same as the manifest from the
lobby.

Hubblenaut commented:
I changed the Command Support color to make it less jarring.
![grafik](https://user-images.githubusercontent.com/6383576/221419231-e7de42f6-ffbf-4b70-8a7d-21bbcca69ae7.png)

SierraKomodo commented:
Taking prior approval for the 24 hour period since the change was just a header color.
